easybuild/framework/easyblock.py

```diff
@@ -214,6 +214,18 @@ def _init_log(self):
 
         self.log = fancylogger.getLogger(name=self.__class__.__name__, fname=False)
 
+        # deprecate self.log.error in easyblocks, self.log.fatal should be used instead
+        def error(msg, *args, **kwargs):
```

Comment:
I agree: if you are going to make breaking changes, then v2.0.0 is a better place for them than a minor release.